pdns/recursordist/syncres.cc

```diff
@@ -4803,6 +4809,11 @@ bool SyncRes::processRecords(const std::string& prefix, const DNSName& qname, co
       */
       if (newtarget.empty() && putInNegCache) {
         g_negCache->add(ne);
+        // doCNAMECacheCheck() checks record cache and does not look into negcache. That means that and old record might be found if
```

Comment:
```suggestion
        // doCNAMECacheCheck() checks record cache and does not look into negcache. That means that an old record might be found if
```
Not important, but "and" -> "an" typo